blocks/init/assets/application-admin.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * This is a main entry point for Application Admin used in `admin`.
```

Comment:
```suggestion
 * This is the main entry point for Application Admin used in `admin`.
 *
```

This intro sounds a bit odd. Maybe replace `admin` with WordPress admin screen?